bin/update_dev_data.py

```diff
@@ -128,14 +128,24 @@ def _update_id_sequence(self, model, value):
 
 
 def devdata():
-    with bootstrap(sys.argv[1]) as env:
+    parser = argparse.ArgumentParser(
+        prog="devdata", description="Inserts standard dev data into the DB"
+    )
+    parser.add_argument("config", help="Location of the paster configuration file")
+    parser.add_argument(
+        "-u",
+        "--repo-url",
+        default="https://github.com/hypothesis/devdata.git",
+        help="Location of the devdata repository",
+    )
```

Comment:
^ Normally I'd put the parser definition at the top as it forms good documentation for the arguments of the script.